src/odemis/gui/cont/tabs.py

```diff
@@ -339,7 +339,6 @@ def get_display_priority(cls, main_data):
 AUTOFOCUS_BINNING = (8, 8)
 AUTOFOCUS_HFW = 300e-06  # m
 
-
 class LocalizationTab(Tab):
```

Comment:
It's a bit of nitpicking but I think we should keep the line break before the class definition as PEP-8 states: https://peps.python.org/pep-0008/#blank-lines